SaintCoinach/ex.json

```diff
@@ -7086,15 +7090,19 @@
         },
         {
           "index": 5,
-          "name": "CustomTalk",
+          "name": "CustomTalk[1]",
```

Comment:
You sure this one is right?  It doesn't appear to have any valid links back to CustomTalk.  Not sure about index 6.